source/manuals/accessibility.html.md.erb

```diff
@@ -84,7 +84,7 @@ If a service is built using an existing or third party system, such as a content
 
 Services should be tested by real users to evaluate how accessible they are. This can be with members of the public as well as specialist organisations who provide accessibility testing services.
 
-You can test your service using an [accessibility empathy lab](https://gds.blog.gov.uk/2018/06/20/creating-the-uk-governments-accessibility-empathy-lab/), such as the one at GDS, which is designed to provide a simulation of using the web with accessibility issues so developers can better understand what this is like.
+You can test your service using an [accessibility lab](https://gds.blog.gov.uk/2018/06/20/creating-the-uk-governments-accessibility-empathy-lab/), such as the one in the GDS London office, which is designed to provide a simulation of using the web with accessibility issues so developers can better understand what this is like.
```

Comment:
The change is fine. But while I read this section I noticed that it only describes half of what the accessibility lab does. The other half is providing all the assistive technologies that are mentioned higher up (JAWS, Dragon, etc).